src/scripts/main.js

```diff
@@ -355,6 +355,21 @@ const people = [
 ];
 
 // eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const dashboard = document.querySelector('.dashboard');
 
-// write your code here
+const visiblePeople = people.map((person) => ({
+  name: person.name,
+  sex: person.sex,
+  born: person.born,
+  died: person.died,
+  age: person.died - person.born,
+  century: Math.ceil(person.died / 100),
+}));
+
+for (let i = 0; i < visiblePeople.length; ++i) {
+  const row = dashboard.insertRow();
+
+  for (const c of Object.keys(visiblePeople[i])) {
+    row.insertCell().textContent = visiblePeople[i][c];
+  }
+}
```

Comment:
Instead of using a for loop to iterate over `visiblePeople`, you could use the `forEach` method. This would make your code cleaner and more readable. Here is how you could do it: `visiblePeople.forEach(person => {...})`.